src/tools/pub/cmd/pub-outdated.md

```diff
@@ -0,0 +1,251 @@
+---
+title: pub outdated
+description: Use pub outdated to help you update your package dependencies.
+diff2html: true
+---
+
+_Outdated_ is one of the commands of the [pub tool](/tools/pub/cmd).
+
+{% prettify nocode tag=pre+code %}
+$ pub outdated [options]
+{% endprettify %}
+
+Use `pub outdated` to identify out-of-date [package dependencies][]
+and get advice on how to update them.
+<span style="color:red">[PENDING: Should we motivate why you want to do this?
+Null safety might be the biggest near-term motivation,
+but are there other major reasons
+that make it worth the effort to update and test your code?]</span>
+
+[package dependencies]: /tools/pub/dependencies
+
+Here's an overview of how you can use `pub outdated`
+to help you update package dependencies:
+
+1. In the top directory of your app or package,
```

Comment:
I'd recommend running outdated before upgrade to see what the status is (similar to the blog post)

Reply:
I started out doing that, but it seemed a little silly to make them run outdated first, when they were almost certainly going to have to run upgrade.

Reply:
Could just be me, but I find it helpful to see the inventory before making any changes to current versions. Helps understand the total set of packages that need updating, and which you should be on the lookout for new issues in.